inference-engine/tests/ie_test_utils/functional_test_utils/include/functional_test_utils/path_utils.hpp

```diff
@@ -0,0 +1,49 @@
+// Copyright (C) 2020 Intel Corporation
```

Comment:
Can all this functionality be in the `file_utils.hpp`? Why we need to have new file?

Reply:
These utils provide different functionality. I think `fileutils` is incorrect place for it